lib/workers/repository/stats.ts

```diff
@@ -5,6 +5,7 @@ import type { RequestStats } from '../../util/http/types';
 
 export function printRequestStats(): void {
   const httpRequests = memCache.get<RequestStats[]>('http-requests');
+  // istanbul ignore next
```

Comment:
What lead to this being necessary?

Reply:
I think I'll better rewrite `renovateRepository()` test with separate PR rather than delete it here. For now it relies on `fs.ensureDir(config.localDir)` throwing exception for uninitialized `localDir` 🙃